ballerina/client.bal

```diff
@@ -13,3 +16,1364 @@
+        http:Request request = new;
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # Redact a conversation part
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - Redact a conversation part 
+    resource isolated function post conversations/redact(RedactConversationRequest payload, RedactConversationHeaders headers = {}) returns Conversation|error {
+        string resourcePath = string `/conversations/redact`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # Search conversations
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - successful 
+    resource isolated function post conversations/search(SearchRequest payload, SearchConversationsHeaders headers = {}) returns ConversationList|error {
+        string resourcePath = string `/conversations/search`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # Create a data attribute
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - Successful 
+    resource isolated function post data_attributes(CreateDataAttributeRequest payload, CreateDataAttributeHeaders headers = {}) returns DataAttribute|error {
+        string resourcePath = string `/data_attributes`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # Submit a data event
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - successful 
+    resource isolated function post events(CreateDataEventRequest payload, CreateDataEventHeaders headers = {}) returns http:Response|error {
+        string resourcePath = string `/events`;
+        map<string|string[]> httpHeaders = getMapForHeaders(headers);
+        http:Request request = new;
+        json jsonBody = payload.toJson();
+        request.setPayload(jsonBody, "application/json");
+        return self.clientEp->post(resourcePath, request, httpHeaders);
+    }
+
+    # Create event summaries
+    #
+    # + headers - Headers to be sent with the request 
+    # + return - successful 
+    resource isolated function post events/summaries(CreateDataEventSummariesRequest payload, DataEventSummariesHeaders headers = {}) returns http:Response|error {
```

Comment:
@PasinduDilmin, Here we have a slight improvement to do the current generated code with `http:Response`, this should be `error?`.

```ballerina
resource isolated function post events/summaries(CreateDataEventSummariesRequest payload, DataEventSummariesHeaders headers = {}) returns error? {
```

Step you can proceed :
Check the operation response which has no content type [1] and modify it by removing other error response code[2].

Ex:[1] Here you can see the success payload is `200` and it has no content type like status code `401`
```json
"/events/summaries": {
        "post": {
          "summary": "Create event summaries",
          "parameters": [
            {
              "name": "Intercom-Version",
              "in": "header",
              "schema": {
                "$ref": "#/components/schemas/intercom_version"
              }
            }
          ],
          "tags": [
            "Data Events"
          ],
          "operationId": "dataEventSummaries",
          "description": "Create event summaries for a user. Event summaries are used to track the number of times an event has occurred, the first time it occurred and the last time it occurred.\n\n",
          "responses": {
            "200": {
              "description": "successful"
            },
            "401": {
              "description": "Unauthorized",
              "content": {
                "application/json": {
                  "examples": {
                    "Unauthorized": {
                      "value": {
                        "type": "error.list",
                        "request_id": "110801d1-2f7b-436f-8f03-2245545a1432",
                        "errors": [
                          {
                            "code": "unauthorized",
                            "message": "Access Token Invalid"
                          }
                        ]
                      }
                    }
                  },
                  "schema": {
                    "$ref": "#/components/schemas/error"
                  }
                }
              }
            }
          }
          }
```

[2] Modified one: remove error response
```json
"post": {
          "summary": "Create event summaries",
          "parameters": [
            {
              "name": "Intercom-Version",
              "in": "header",
              "schema": {
                "$ref": "#/components/schemas/intercom_version"
              }
            }
          ],
          "tags": [
            "Data Events"
          ],
          "operationId": "dataEventSummaries",
          "description": "Create event summaries for a user. Event summaries are used to track the number of times an event has occurred, the first time it occurred and the last time it occurred.\n\n",
          "responses": {
            "200": {
              "description": "successful"
                }                
              }
            }
          
```
***Since we currently concern with the success response , it is fine to remove error response

Shall we update the OpenAPI spec with this detail in the applicable operations, and regenerate the client? and update the sanitization documentation too?